native_client/setup.py

```diff
@@ -4,13 +4,19 @@
 from distutils.command.build import build
 
 import os
-import numpy
 import subprocess
 
 try:
-    numpy_include = numpy.get_include()
-except AttributeError:
-    numpy_include = numpy.get_numpy_include()
+    import numpy
```

Comment:
We should not require `numpy` there, because we might indeed install the host version when trying to build the target's version.